docs/make.bat

```diff
@@ -0,0 +1,263 @@
+@ECHO OFF

```

Comment:
@emanueldima fixed. I also added the standard files like AUTHORS, and LICENSE.

I created a CONTRIBUTING.rst file as github automatically points to it when users create new issues. It is an enhanced version of what the wiki has. We can fix the wiki when the evolution branch replaces the master branch.

The `*.rst` files, including the README which I converted from markdown, are now included in the generated documentation.

This documentation can then be automatically published when B2Share will be released in Pypi.